pandas/core/frame.py

```diff
@@ -5209,8 +5206,8 @@ def swaplevel(self, i=-2, j=-1, axis=0):
 
         Parameters
         ----------
-        i, j : int, str (can be mixed)
-            Level of index to be swapped. Can pass level name as string.
+        i, j : int or str
+            Levels indices to be swapped. Can pass level name as string.
```

Comment:
```suggestion
            Levels of the indices to be swapped. Can pass level name as string.
```
Clearer and consistent with the other docstrings.